### PR TITLE
Release 2.0.1

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

- https://github.com/mattermost/mattermost-plugin-jitsi/pull/224
- https://github.com/mattermost/mattermost-plugin-jitsi/pull/214

#### Ticket Link

https://github.com/mattermost/mattermost-plugin-jitsi/issues/225

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -18,16 +18,7 @@ require (
...
-	golang.org/x/net v0.0.0-20200528225125-3c3fba18258b // indirect
-	golang.org/x/sys v0.0.0-20200523222454-059865788121 // indirect
+	github.com/stretchr/testify v1.6.1
+	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a // indirect
 	golang.org/x/text v0.3.2
-	golang.org/x/tools v0.0.0-20200528185414-6be401e3f76e // indirect
-	google.golang.org/genproto v0.0.0-20200528191852-705c0b31589b // indirect
-	google.golang.org/grpc v1.29.1 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20200506231410-2ff61e1afc86 // indirect
-	honnef.co/go/tools v0.0.1-2020.1.4 // indirect
```
Comment (mickmister) [inline go.mod:32]: Not sure about these changes

**Review by hanzei — APPROVED** [review]
LGTM

**Comment by hanzei** [comment]
https://github.com/mattermost/mattermost-plugin-jitsi/pull/227 got merged

**Review by jespino — APPROVED** [review]
LGTM

**Pull request merged**